src/client/AM.PA.MonitoringTool/ActivityMapper/ProcessToActivityMapper.cs

```diff
@@ -175,7 +190,7 @@ private static ActivityCategory GetCategory(string processName, string windowNam
             }
             catch (Exception e)
             {
-                Console.WriteLine("> ERROR while Mapping: {0}", e.Message);
+                //Console.WriteLine("> ERROR while Mapping: {0}", e.Message);
```

Comment:
Why not log this?
Should this be treated in any way?

Reply:
You don't have a console window open in a wpf application. This usually goes nowhere (except in the debugger I think). Viable alternatives are Trace.WriteLine, Debug.WriteLine or some other logging method. This is why I only commented it out.

Reply:
@casaout How would you handle these errors? Do you have a log file?

Reply:
Depending on the error, we log issues into the log-table in the database. Database.GetInstance().LogError(…) -- Like that you can also log warnings and Infos.

For crazy things that make the tool/tracker unusable, we log them in the logfile Logger.WriteToLog(e); // here you can just add the exception directly